po/R-fr.po

```diff
@@ -0,0 +1,269 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: pkgdown 1.9000.9000.9000\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-10-28 08:02-0500\n"
+"PO-Revision-Date: 2021-10-27 11:57-0500\n"
+"Last-Translator: \n"
+"Language-Team: fr\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: potools 0.2.2\n"
+
+#: build-404.R:11
+msgid "Page not found (404)"
+msgstr "Page introuvable (404)"
+
+#: build-404.R:12
+msgid "Content not found. Please use links in the navbar."
+msgstr "Contenu introuvable. Veuillez utiliser les liens de la navbar."
+
+#: build-articles.R:371 navbar.R:134 navbar.R:141 navbar.R:156
+msgid "Articles"
+msgstr "Articles"
+
+#: build-footer.R:29
+msgid "Developed by"
+msgstr "Développé par"
+
+#: build-footer.R:34
+msgid "Site built with <a href=\"%s\">pkgdown</a> %s."
+msgstr "Site créé avec <a href=\"%s\">pkgdown</a> %s."
+
+#: build-home-authors.R:92
+msgid "More about authors..."
+msgstr "À propos des auteurs..."
+
+#: build-home-authors.R:95
+msgid "Developers"
+msgstr "Développeurs"
+
+#: build-home-authors.R:108 render.R:126
+msgid "Authors"
+msgstr "Auteurs"
+
+#: build-home-authors.R:200
+msgid "author"
+msgstr "auteur"
```

Comment:
I wonder whether this should be "auteur·trice". (same for other role names)

Reply:
Let me merge @jplecavalier's suggestions, and then I'd really appreciate the inclusive French equivalents for these terms.

Reply:
Totally agree with @maelle on that, good catch